lib/io.toit

```diff
@@ -1587,17 +1629,17 @@ class EndianBuffer extends EndianWriter:
 
   /** Writes the given int64 $value to this buffer at the given index $at. */
   put-int64 --at/int value/int:
-    endian_.put-int64 cached-byte-array_ at value
+    byte-order_.put-int64 cached-byte-array_ at value
     buffer_.put --at=at cached-byte-array_ 0 8
```

Comment:
This is kinda of a weird pattern. Why don't you just write it into the buffer?

Reply:
The `buffer_` (of type `Buffer`) has a `buffer_` (of type `ByteArray`).
We could inline the `buffer_.put` call. After all, it's just:
```
  // In the `Buffer` class.
  put --at/int data/Data from/int=0 to/int=data.byte-size:
    if not 0 <= at <= at + (to - from) <= offset_: throw "INVALID_ARGUMENT"
    buffer_.replace at data from to
```

Not sure it's worth it.